src/pages/SignIn.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import { TextField, Button, Typography } from '@material-ui/core';
+import Logo from '../components/Logo';
+import Divider from '@material-ui/core/Divider';
+
+import IconButton from '@material-ui/core/IconButton';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputLabel from '@material-ui/core/InputLabel';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import FormControl from '@material-ui/core/FormControl';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import FacebookButton from '../components/FacebookButton';
+import GoogleButton from '../components/GoogleButton';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    width: '100%',
+    height: '100%'
+  },
+  leftChild: {
+    width: '42%',
+    backgroundColor: theme.palette.primary.dark,
+    '& > *': {
+      color: theme.palette.common.white
+    },
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    '& > p': {
+      margin: theme.spacing(20, 18, 20, 20),
+      fontSize: '1.8rem'
+    }
+  },
+  rightChild: {
+    width: '58%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    margin: theme.spacing(0, 0, 11, 0)
+  },
+  buttons: {
+    display: 'flex',
+    flexDirection: 'column',
+    '& > *': {
+      margin: theme.spacing(1, 0)
+    }
+  }
+}));
+
+function SignIn(props) {
+  const classes = useStyles();
+  const [email, setEmail] = useState('');
+  const [values, setValues] = React.useState({
+    amount: '',
```

Comment:
amount, weight and weightRange aren't being used anywhere. Double check code examples you copied when you use them